### ISSUE TITLE
bug:  Problem with gulp-sass and node 4.1.1

### ISSUE THREAD

**Issue by intrepido** [driftyco/ionic#4462]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>This is my enviroment with the command `ionic info`
```
Your system information:

Cordova CLI: 5.3.3
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.0
Ionic Version: 1.1.0
Ionic CLI Version: 1.6.5
Ionic App Lib Version: 0.3.9
OS: Windows 8
Node Version: v4.1.1
```
When I run the command `npm install`, I get the following error:

> node-sass@2.1.1 install C:\Users\FIDEL-CASA\Desktop\prueba\dsfasdfa\fidel\node
_modules\gulp-sass\node_modules\node-sass
> node scripts/install.js

> Can not download file from https://raw.githubusercontent.com/sass/node-sass-binaries/v2.1.1/win32-x64-node-4.1/binding.node
> 

> node-sass@2.1.1 postinstall C:\Users\FIDEL-CASA\Desktop\prueba\dsfasdfa\fidel\
node_modules\gulp-sass\node_modules\node-sass
> node scripts/build.js

> gyp WARN install got an error, rolling back install
> gyp ERR! configure error
> gyp ERR! stack Error: 404 status code downloading 64-bit node.lib
> gyp ERR! stack     at Request.<anonymous> (C:\Users\FIDEL-CASA\Desktop\prueba\ds
> fasdfa\fidel\node_modules\gulp-sass\node_modules\node-sass\node_modules\pangyp\l
> ib\install.js:403:20)
> gyp ERR! stack     at emitOne (events.js:82:20)
> gyp ERR! stack     at Request.emit (events.js:169:7)
> gyp ERR! stack     at Request.onRequestResponse (C:\Users\FIDEL-CASA\Desktop\pru
> eba\dsfasdfa\fidel\node_modules\gulp-sass\node_modules\node-sass\node_modules\pa
> ngyp\node_modules\request\request.js:1255:10)
> gyp ERR! stack     at emitOne (events.js:77:13)
> gyp ERR! stack     at ClientRequest.emit (events.js:169:7)
> gyp ERR! stack     at HTTPParser.parserOnIncomingClient (_http_client.js:415:21)

> gyp ERR! stack     at HTTPParser.parserOnHeadersComplete (_http_common.js:88:23)

> gyp ERR! stack     at Socket.socketOnData (_http_client.js:305:20)
> gyp ERR! stack     at emitOne (events.js:77:13)
> gyp ERR! System Windows_NT 6.2.9200
> gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\FIDEL-CASA\\D
> esktop\\prueba\\dsfasdfa\\fidel\\node_modules\\gulp-sass\\node_modules\\node-sas
> s\\node_modules\\pangyp\\bin\\node-gyp" "rebuild"
> gyp ERR! cwd C:\Users\FIDEL-CASA\Desktop\prueba\dsfasdfa\fidel\node_modules\gulp
> -sass\node_modules\node-sass
> gyp ERR! node -v v4.1.1
> gyp ERR! pangyp -v v2.3.2
> gyp ERR! not ok
> Build failed
> npm ERR! Windows_NT 6.2.9200
> npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
> node_modules\\npm\\bin\\npm-cli.js" "install"
> npm ERR! node v4.1.1
> npm ERR! npm  v2.14.4
> npm ERR! path C:\Users\FIDEL-CASA\AppData\Roaming\npm-cache\lodash.template\3.6.
> 2\package\package.json.85c499c6dffff9035680b8e91062bf92
> npm ERR! code EPERM
> npm ERR! errno -4048
> npm ERR! syscall rename

> npm ERR! Error: EPERM: operation not permitted, rename 'C:\Users\FIDEL-CASA\AppD
> ata\Roaming\npm-cache\lodash.template\3.6.2\package\package.json.85c499c6dffff90
> 35680b8e91062bf92' -> 'C:\Users\FIDEL-CASA\AppData\Roaming\npm-cache\lodash.temp
> late\3.6.2\package\package.json'
> npm ERR!     at Error (native)
> npm ERR!  { [Error: EPERM: operation not permitted, rename 'C:\Users\FIDEL-CASA\
> AppData\Roaming\npm-cache\lodash.template\3.6.2\package\package.json.85c499c6dff
> ff9035680b8e91062bf92' -> 'C:\Users\FIDEL-CASA\AppData\Roaming\npm-cache\lodash.
> template\3.6.2\package\package.json']
> npm ERR!   errno: -4048,
> npm ERR!   code: 'EPERM',
> npm ERR!   syscall: 'rename',
> npm ERR!   path: 'C:\\Users\\FIDEL-CASA\\AppData\\Roaming\\npm-cache\\lodash.tem
> plate\\3.6.2\\package\\package.json.85c499c6dffff9035680b8e91062bf92',
> npm ERR!   dest: 'C:\\Users\\FIDEL-CASA\\AppData\\Roaming\\npm-cache\\lodash.tem
> plate\\3.6.2\\package\\package.json',
> npm ERR!   parent: 'gulp-util' }
> npm ERR!
> npm ERR! Please try running this command again as root/Administrator.

> npm ERR! Please include the following file with any support request:
> npm ERR!     C:\Users\FIDEL-CASA\Desktop\prueba\dsfasdfa\fidel\npm-debug.log

I think the problem is caused by the version of `"gulp-sass": "^1.3.3"` which uses the ```"node-sass" : "^2.0.1"```. Here https://goo.gl/jFeu7s said that `node-sass` version does not work with node v4.1.1, it does not work even with the node v4.0.0. 

Do you recommend in this case back to node v0.12.7 ? 
Why Ionic not use a more recent gulp-sass version to resolve that issue?

</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @intrepido!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4462). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4462)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by micrdy** [comment]
Same here on OSX. Gonna downgrade node...

**Comment by AlexSwensen** [comment]
Change your gulp-sass dependency to the current version (mine is on ^2.0.4)  in your package.json and then do an npm install. This will fix your issue.

Tested on Node 4.1.1

**Comment by kylefox** [comment]
I also encountered this and solved it by specifying `"gulp-sass": "^2.0.4",` in `package.json`.

Possibly a duplicate of https://github.com/driftyco/ionic/issues/4434?

**Comment by mhartington** [comment]
Not a duplicate, but along the same lines as #4434.

Fixing this tomorrow.
